ot3_state_manager/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "ot3_state_manager"
+version = "0.1.0"
+description = ""
+authors = ["Derek Maggio <derek.maggio@opentrons.com>"]
```

Comment:
we tend to make our authors `Opentrons <engineering@opentrons.com>` i believe

Reply:
https://github.com/Opentrons/ot3-firmware/pull/418/commits/7d71df09720e545a05b7d9d7ef1b08717593ec2f